addon/helpers/page-title.js

```diff
@@ -16,8 +16,9 @@ export default Ember.Helper.extend({
     tokens.push({ id: guidFor(this) });
   },
 
-  compute(params, hash) {
+  compute(params, _hash) {
     let tokens = get(this, 'pageTitleList');
+    let hash = Ember.merge({}, _hash);
```

Comment:
@Turbo87 - The rest of this file uses destructuring, can you update this to follow suit?